ui-guidelines/css/guidelines.css

```diff
@@ -165,8 +165,15 @@ pre {
 	font-size: 1em;
 	font-weight: bold;
 }
-/*==== HELPERS ====*/
-.top-half-extra {margin-top: 2em;}
-.top-extra {margin-top: 4em;}
-.top-low {margin-top: 1.5em;}
-.bottom-half-extra {margin-bottom: 1.5em;}
\ No newline at end of file
+
+/*==== UTILITIES ====*/
+.u-space-top-big {margin-top: 4em;}
+.u-space-top-xlarge {margin-top: 2em;}
+.u-space-top-large {margin-top: 1.5em;}
+.u-space-top {margin-top: 1em;}
+.u-space-bottom-large {margin-bottom: 1.5em;}
+.u-space-left-xlarge {margin-left: 2em;}
+.u-move-top-xlarge {margin-top: -2em;}
```

Comment:
`.u-move` might be a bit misleading, because `.u-space` also moves the element.
What do you think to call them `.u-move-up` and `.u-move-down`?